elasticdl/pkg/common/tensor.go

```diff
@@ -0,0 +1,64 @@
+package common
+
+import (
+	"bytes"
+	"elasticdl.org/elasticdl/pkg/proto"
+	"encoding/binary"
+	"math"
+)
+
+// Tensor defines tensor struct
+// TODO(qijun): handle different tensor dtype
+type Tensor struct {
+	Name    string
+	Value   []float32
+	Dim     []int64
+	Indices []int64
+}
+
+// GetDimProduct returns the product of a slice of dim
+func GetDimProduct(dim []int64) int64 {
```

Comment:
Will `GetDimProduct` be used outside this file? If not, change it to an unexported func. If yes, keep as it is.

Reply:
I am not very sure. 
We could set it to an unexported func first. If it is used in another place, we change it.